Src/Screens/ProfileScreen/Screens/MyPayment/MyPayment.swift

```diff
@@ -18,12 +18,26 @@ struct MyPayment {
 extension MyPayment: Presentable {
     func materialize() -> (UIViewController, Disposable) {
         let bag = DisposeBag()
+        
+        let dataSignal = self.client.watch(query: MyPaymentQuery()).map { $0.data }
+        let failedChargesSignalData = dataSignal.map { $0?.balance.failedCharges }
+        let nextPaymentSignalData = dataSignal.map { $0?.nextChargeDate }
 
         let viewController = UIViewController()
         viewController.title = String(key: .MY_PAYMENT_TITLE)
 
         let form = FormView()
         bag += viewController.install(form)
+        
+        bag += combineLatest(failedChargesSignalData, nextPaymentSignalData).onValue({ failedCharges, nextPayment in
```

Comment:
use `onValueDisposePrevious` and return the disposable from L38, so that we don't create multiple warnings when the data reloads 😄